app/_how-tos/configure-oidc-with-auth-code-flow.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Configure OpenID Connect with the authorization code flow
+content_type: how_to
+
+related_resources:
+  - text: Authentication in {{site.base_gateway}}
+    url: /gateway/authentication/
+  - text: OpenID Connect authentication flows and grants
+    url: /plugins/openid-connect/#authentications
+  - text: Authorization code workflow
+    url: /plugins/openid-connect/#authorization-code-flow
+
+plugins:
+  - openid-connect
+
+entities:
+  - route
+  - service
+
+products:
+  - gateway
+
+works_on:
+  - on-prem
+  - konnect
+
+min_version:
+  gateway: '3.4'
+
+tools:
+  - deck
+
+prereqs:
+  entities:
+    services:
+      - example-service
+    routes:
+      - example-route
+  inline:
+    - title: Set up Keycloak
+      include_content: prereqs/auth/oidc/keycloak-password
+      icon_url: /assets/icons/keycloak.svg
+
+tags:
+  - authentication
+  - openid-connect
+
+tldr:
+  q: How do I use an authorization code to open a session with my identity provider, letting users log in through a browser?
+  a: Using the OpenID Connect plugin, set up the [auth code flow](/plugins/openid-connect/#authorization-code-flow) to connect to an identity provider (IdP) through a browser, and use session authentication to store open sessions.
+
+cleanup:
+  inline:
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg
+
+---
+
+## 1. Enable the OpenID Connect plugin with the auth code flow
+
+Using the Keycloak and {{site.base_gateway}} configuration from the [prerequisites](#prerequisites), 
+set up an instance of the OpenID Connect plugin with the auth code flow and session authentication.
+These two 
```

Comment:
This sentence is incomplete. I wasn't sure what it was trying to say, so I couldn't provide a suggestion.